src/tokens/selectors.ts

```diff
@@ -366,6 +368,7 @@ export const swappableToTokensByNetworkIdSelector = createSelector(
     return tokens.filter(
       (tokenInfo) =>
         tokenInfo.isSwappable ||
+        tokenInfo.isManuallyImported ||
```

Comment:
i think this change is also in your other open PR, but do we need this line when it is already in the `swappableFromTokensByNetworkIdSelector`?

Reply:
Yep, this code is twice since I need it to test.
I [answered in the other PR](https://github.com/valora-inc/wallet/pull/4930#discussion_r1499496578)

Reply:
oooohhhh no sorry you're totally right, i flipped the intention of the /filter yesterday. this is fine 👌🏼